ffv1.md

```diff
@@ -1292,6 +1292,63 @@ The reference implementation [@REFIMPL] was validated in the following condition
 
 In all of the conditions above, the decoder and encoder was run inside the [@VALGRIND] memory debugger as well as clangs address sanitizer [@Address-Sanitizer], which track reads and writes to invalid memory regions as well as the use of uninitialized memory.  There were no errors reported on any of the tested conditions.
 
+# Media Type Definition
+
+This registration is done using the template defined in [@!RFC6838] and following [@!RFC4855].
+
+Type name:  video
+
+Subtype name:  FFV1
+
+Required parameters:  None.
+
+Optional parameters:
+
+  This parameter is used to signal the capabilities of a receiver implementation. This parameter MUST NOT be used for any other purpose.
+
+  version:  The version of the FFV1 encoding as defined by [in the section on `version`](#version).
```

Comment:
I would put version\[.micro_version\].

Similar to https://www.iana.org/assignments/media-types/video/H264 parameters, I would add coder_type, color_space and bits_per_raw_sample, as some decoders may support only some combinations or we may add new coder_type, color_space in the future (like H.264 decoders support only some profiles and/or levels and standard adds new profiles later). Adding chroma subsampling info and others is maybe too much?

Reading IANA consideration for H264, max-fs (frame size), max-br (bit rate), something about max count of slices, could be taken into consideration.

Reply:
the pr is updated to consider this comment. please re-review.

Reply:
Fine for me but willing another pair of eyes on it, I am not confident enough that I saw everything.

Reply:
thx, I suspect we'll get a better review when reviewed by the area director or iesg, but for now I hope we have consensus amongst participating contributors/reviewers.